OpenMATB.py

```diff
@@ -888,16 +888,16 @@ def getConfigValue(key, defaultvalue):
     app = QtWidgets.QApplication(sys.argv)
 
     loadConfig()
+    #scenario_FullPath, none = QtWidgets.QFileDialog.getOpenFileName(
+    #    None, VERSIONTITLE + ' - ' + _('Select a scenario'), SCENARIOS_PATH, "(*.txt)")
 
-    scenario_FullPath, none = QtWidgets.QFileDialog.getOpenFileName(
-        None, VERSIONTITLE + ' - ' + _('Select a scenario'), SCENARIOS_PATH, "(*.txt)")
-
+    scenario_FullPath = '/home/jon/nasa_tlx/OpenMATB/Scenarios/plant.txt'
```

Comment:
Can we find a better solution that this software can be started on any laptop?

Reply:
yes the file browser is now running

Reply:
nice!